backend/src/test/java/pl/krkteam/battleships/common/domain/GameBoardHolderTest.java

```diff
@@ -87,4 +92,39 @@ public void shouldReturnFalseWhenGivenPlayerToAddAlreadyExists() {
         // when - then
         Assert.assertFalse(addResult);
     }
+
+    @Test
+    public void testAreBothFleetsValidAndExpectAreValid() {
+        GameBoardHolder gameBoardHolder = new GameBoardHolder();
+        gameBoardHolder.addPlayer(player1, gameBoard1);
+        gameBoardHolder.addPlayer(player2, gameBoard2);
+
+        when(gameBoard1.isPlacedFleet()).thenReturn(true);
+        when(gameBoard2.isPlacedFleet()).thenReturn(true);
+
+        assertEquals(gameBoardHolder.areBothFleetsValid(player1).getResult(),
+                new ReadyForPlayDTO().getResult());
+    }
+
+    @Test
+    public void testAreBothFleetsValidAndExpectAreNotValid() {
+        GameBoardHolder gameBoardHolder = new GameBoardHolder();
```

Comment:
I would extract private method from this, like initGameboard.

Reply:
Does it make any sense?!